### PR TITLE
Add command to import osu_scores_high tables into solo_scores

### PR DESCRIPTION
**Pull request by smoogipoo**
- [x] https://github.com/ppy/osu/pull/15657

This is highly WIP, and shouldn't be merged before we discuss how to deal with the PKs and all.

It can be used for testing other score statistics processors as it is.

### CODE REVIEW

**Comment by smoogipoo** [comment]
What `build_id` should scores from stable be imported with? Currently it imports with `null`. Will there be a special build providing a `build_id`?

**Comment by peppy** [comment]
There are already `build_id`s for every stable build, not sure what to do about those that don't have that info though.

**Comment by smoogipoo** [comment]
Yeah the question is which one to pick.

**Comment by peppy** [comment]
I'd leave null if undetermined.

**Comment by peppy** [comment]
@smoogipoo does this still need to be blocked? i think we can get it merged and work from there?

File: osu.Server.Queues.ScorePump/ImportHighScores.cs [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+                            created_at = highScore.date,
+                            updated_at = highScore.date
+                        },
+                        created_at = highScore.date,
+                        updated_at = highScore.date,
+                    };
+
+                    // Todo: Import highScore.hidden somehow?
+
+                    using (var transaction = db.BeginTransaction())
```
Comment (peppy) [inline osu.Server.Queues.ScorePump/ImportHighScores.cs:95]: Definitely don't want one transaction per insert, but can fix that when we get closer to running this on mass scores.

**Review by peppy — APPROVED** [review]

**Comment by smoogipoo** [comment]
Doesn't need to be blocked, no, if you're fine with it as is.

File: osu.Server.Queues.ScorePump/ImportHighScores.cs (hunk 1 above)
Comment (smoogipoo) [reply osu.Server.Queues.ScorePump/ImportHighScores.cs:96]: One thing to note is that currently the previous score id is stored inside the json data. So right now `score.score_id != score.data.score_id` for these particular scores.

**Pull request merged**